### ISSUE TITLE
[Feature Request] Attack Phases

### ISSUE THREAD

**Issue by Darkosto** [Lothrazar/PlayerBosses#3]
Hey!

After thinking about it for a few days, it would be pretty cool to give the boss the ability to switch attack types at specified health thresholds. Right now, the bosses will only do melee. What if the boss switched to these modes at 1/3 health segments:

100-66 Melee (current iteration of mod)
65-33 Skeleton-like ranged attacks (grabs a bow?)
32-0 Ghast-like attacks with fireballs

Thanks for considering this!
Darkosto

**Comment by Lothrazar** [comment]
Yep that should be possible.  I need to figure out how to remove and add AI behavior, and remember the current state, but after that its easy.  Ill get back to you in a few days or so

**Comment by Lothrazar** [comment]
WHen i did melee first then bow i couldnt get the ai to switch.  

So it will be more like

Bow to start -> then melee just like befrore -> , then ghast fireballs.  

Each health checkpoint in config as a percentage

**Comment by Darkosto** [comment]
Sounds like a good plan to me! Thank you :)

**Comment by Lothrazar** [comment]
i fought it a few times in survival with unenchanted diamond gear and had no problem in default settings  , seems ok.   

Threw it up so hopefully no issues https://www.curseforge.com/minecraft/mc-mods/giant-player-boss/files/2733042
